aws/src/main/java/org/apache/iceberg/aws/glue/GlueCatalog.java

```diff
@@ -151,7 +151,12 @@ private LockManager initializeLockManager(Map<String, String> properties) {
     if (properties.containsKey(CatalogProperties.LOCK_IMPL)) {
       return LockManagers.from(properties);
     } else if (SET_VERSION_ID.isNoop()) {
+      LOG.warn(
+          "Optimistic locking is not available in the environment. Using in-memory lock manager."
+              + " To ensure atomic transaction, you need to setup a DynamoDB lock manager.");
```

Comment:
Nit: generally in log messages I think we try to avoid the second person. 

"To ensure atomic transaction, please configure a distributed lock manager such as the DynamoDB lock manager"

Reply:
Fixed the warn message as you suggested. Thank you @amogh-jahagirdar